README.md

```diff
@@ -107,6 +107,17 @@ The Kotlin language server supports some non-standard requests through LSP. See
 ### Global symbols
 ![Global symbols](images/GlobalSymbols.png)
 
+## Troubleshooting
+### java.lang.OutOfMemoryError
```

Comment:
Wdyt about putting this into the `TROUBLESHOOTING.md` instead?

Reply:
Did not notice that one. Moved it there now, and created a link in the readme. That file really needs a link... 😞